addon/styles/addon.scss

```diff
@@ -3,89 +3,74 @@
 
 $z-index-application-bar: 3 !default;
 $z-index-modal: 2 !default;
+$z-index-modal-hidden: -1 !default;
 
 .frost-application-bar {
+  background-color: $frost-color-night-2;
+  display: flex;
+  height: 50px;
+  left: 0;
   position: fixed;
   top: 0;
-  left: 0;
-  height: 50px;
   width: 100%;
-  display: flex;
-  background-color: $frost-color-night-2;
   z-index: $z-index-application-bar;
 
   .branding {
-    height: 50px;
-    padding: 0 25px;
+    align-items: center;
     border-right: 1px solid $frost-color-grey-3;
     display: flex;
-    align-items: center;
+    height: 50px;
+    padding: 0 25px;
   }
 
   .navigation {
-    padding: 0 15px;
-    height: 50px;
     border-right: solid 1px $frost-color-grey-3;
+    height: 50px;
+    padding: 0 15px;
 
     .frost-icon {
-      width: 35px;
       height: 35px;
       margin-right: 10px;
+      width: 35px;
     }
   }
 }
 
 //TODO import pattern from ember-css-* when it's available
 // import from frost-css patterns modal.scss
-.lf-overlay {
-  top: 50px;
-  opacity: 0.1;
-}
+.nav-modal-binding {
+  max-height: 80vh;
+  min-height: 500px;
 
-.lf-dialog, .lm-container {
-  z-index: $z-index-modal;
-}
-
-.frost-navigation {
-  display: flex;
-  flex-direction: row;
-
-  & > div:first-of-type {
-    border-bottom: 1px solid $frost-color-grey-2;
-    border-left: 1px solid rgba($frost-color-white, 0.2);
-    overflow: hidden;
-    .user {
-      border-left: none;
+  position: fixed;
+  width: 100%;
+  &.liquid-container {
+    overflow: inherit;
+    z-index: $z-index-modal;
+    .is_hidden & {
+      &:not(.liquid-animating) {
+        z-index: $z-index-modal-hidden;
```

Comment:
I think that's only the case because the `to-elsewhere` isn't conditional currently